supportedBackends/jes/src/test/scala/cromwell/backend/impl/jes/JesInitializationActorSpec.scala

```diff
@@ -132,7 +132,7 @@ class JesInitializationActorSpec extends TestKitSuite("JesInitializationActorSpe
       |}
       | """.stripMargin))
 
-  val defaultBackendConfig = new BackendConfigurationDescriptor(backendConfig, globalConfig)
+  val defaultBackendConfig = BackendConfigurationDescriptor(backendConfig, globalConfig)
```

Comment:
While you're here, might as well go ahead and make `BackendConfigurationDescriptor` a `final case class`